tests/lib/options.js

```diff
@@ -356,12 +356,6 @@ describe("options", function() {
 
             assert.equal(currentOptions.parser, "test");
         });
-
-        it("should return a espree if --parser is not passed", function() {
```

Comment:
Is this no problem?
We have an API to get parser's module: http://eslint.org/docs/developer-guide/working-with-rules#the-context-object